### PR TITLE
social media icons on website

### PR DESCRIPTION
**Pull request by bymadisonross**
figuring out shit arent u proud

### CODE REVIEW

File: src/home/templates/home/landing_page.html [hunk 1]
```diff
@@ -31,8 +31,10 @@
                                     <a href="mailto:nellywilson@ufl.edu?Subject=Recruitment%20with%20GatorTech" class="btn btn-default btn-apple"><i class="fa fa-pencil"></i>Recruiters, email us!</a>
                                 </li>
                                 <li>
-                                    <a href="https://docs.google.com/forms/d/e/1FAIpQLSe_P5e7dgGNWFHt386TFam4lz2qFcq2ddNUkWgKRtQBlUphow/viewform" class="btn btn-default btn-apple"><i class="fa fa-usd"></i>Order shirts here!</a>
+                                <a href="https://www.facebook.com/gatortechuf"i class="btn btn-default btn-apple"><i class="fa fa-facebook-official"></i>Stay updated on our Facebook page!</a>
                                 </li>
+                                <li>
+                                    <a href="https://twitter.com/GatorTechUF" class="btn btn-default btn-apple"><i class="fa fa-twitter"></i>Find us on Twitter!</a>
```
Comment (ryansheppard) [inline src/home/templates/home/landing_page.html]: You forgot to add a </li> tag to this list item.

Comment (ryansheppard) [reply src/home/templates/home/landing_page.html]: specifically the `</li>` tag

File: src/home/templates/home/landing_page.html [hunk 2]
```diff
@@ -31,8 +31,10 @@
                                     <a href="mailto:nellywilson@ufl.edu?Subject=Recruitment%20with%20GatorTech" class="btn btn-default btn-apple"><i class="fa fa-pencil"></i>Recruiters, email us!</a>
                                 </li>
                                 <li>
-                                    <a href="https://docs.google.com/forms/d/e/1FAIpQLSe_P5e7dgGNWFHt386TFam4lz2qFcq2ddNUkWgKRtQBlUphow/viewform" class="btn btn-default btn-apple"><i class="fa fa-usd"></i>Order shirts here!</a>
+                                <a href="https://www.facebook.com/gatortechuf"i class="btn btn-default btn-apple"><i class="fa fa-facebook-official"></i>Stay updated on our Facebook page!</a>
```
Comment (ryansheppard) [inline src/home/templates/home/landing_page.html]: There is a random 'i' between `gatortechuf"` and `class`

File: src/home/templates/home/landing_page.html (hunk 1 above)
Comment (bymadisonross) [reply src/home/templates/home/landing_page.html]: ugh. fixing now.

**Comment by bymadisonross** [comment]
I fixed it but unsure if it's showing up here? sos

**Pull request merged**